schema/transportation/segment.yaml

```diff
@@ -51,6 +57,71 @@ properties:
         default: []
 "$defs":
   propertyDefinitions:
+    destinationLabelType:
+      description: >-
+        The type of object of the destination label.
+      type: string
+      enum: 
+      - place
```

Comment:
This is an unfortunate intersection with the places theme.

What OSM tags/combinations does `place` come from, and can we think of something else to name this?

Reply:
renamed to `unknown`